### DISCUSSION TITLE
Show your PixelIt

### DISCUSSION THREAD

**Discussion by o0shojo0o** [pixelit-project/PixelIt#48] [Show and tell]
**Lass mal eure Umsetzungen sehen!** 
Ich fange mal an.... 

![image](https://user-images.githubusercontent.com/19333515/121056078-1ed63300-c7be-11eb-8b93-c3a05bff325f.png)
![image](https://user-images.githubusercontent.com/19333515/121056066-1bdb4280-c7be-11eb-9cce-5aeddee91d08.png)
![1559042250-32757-03-03-2019-23-47-50-9767](https://user-images.githubusercontent.com/19333515/121056272-52b15880-c7be-11eb-8b0f-ec2d96afcc07.jpeg)

Als Diffuser nutze ich eine Milchglasfolie und als Tönung eine Tönungsfolie aus den KFZ-Bereich.

### Holz Gehäuse: 
![image](https://user-images.githubusercontent.com/19333515/121056293-58a73980-c7be-11eb-9865-1980b405467c.png)

Auch hier nutze ich als Diffuser eine Milchglasfolie und als Tönung eine Plexiglasscheibe von [Expresszuschnitt.de Typ:  Acrylglas satiniert grau PERSPEX® Frost](https://expresszuschnitt.de/Acrylglas-satiniert-grau).

**Comment by miccgn** [comment]
Hallo,
ich habe mir das Gehäuse von Thingiverse drucken lassen. Jetzt frage ich mich: wie befestige ich daran bzw. darin am besten die Plexiglasscheibe? Sie lässt sich ja mehr oder weniger bündig vor der Matrix versenken, aber wie bekomme ich sie zum halten? Auf einer Hälfte des Gehäuse kleben, die andere lose lassen, so dass ich ggf. das Gehäuse nochmal öffnen kann? (Was schwer werden dürfte, da die beiden Gehäusehälften gut verzahnt sind.) Oder mit 1-2 Tropfen Heißkleber nur in den Ecken befestigen, so dass ich ggf. mit etwas Hebelkraft das Ganze nochmal aufbekomme?

**Reply by o0shojo0o** [reply]
Hi,
bei mir hält die Plexiglasscheibe tatsächlich allein durch die Klemmwirkung :)

**Reply by miccgn** [reply]
Das hat bei mir nicht geklappt. Die Plexiglasscheibe war dann doch 0,2mm zu schmal dafür.
Ich bastele jetzt eine zweite PixelIt, diesmal vermutlich mit dem AWTrix-Gehäuse (aber der PixelIt-SW), und klebe die erste mit Silikon fest. Ist nur unpraktisch, wenn ich an die SD-Karte muss. Daher werde ich wohl noch eine Sperrdiode zwischen DFPlayer und D1 integrieren und dem DFPlayer einen USB-Anschluss verpassen, der von außen zugänglich ist. Dann kann ich die SD-Karte per USB betanken.

**Comment by rgruebel** [comment]
@o0shojo0o 
Wird die Milchglasfolie auf die Plexiglasscheibe geklebt?
Welche Folie kannst du empfehlen?

**Reply by o0shojo0o** [reply]
Hi, ich habe die Folie auf das Gitter geklebt. 
Ich habe diese Folie genommen da meine Frau die eh da hatte https://kreativmanufaktur.bayern/epages/f41caa8a-1aea-495f-91d6-ef9485e1db47.sf/de_DE/?ObjectPath=/Shops/f41caa8a-1aea-495f-91d6-ef9485e1db47/Products/Milchglasfolie/SubProducts/Milchglasfolie_A4

**Reply by miccgn** [reply]
Ich kann diese empfehlen, die ich schon in verschiedenen Projekten verwendet habe:
https://www.leuchtbildshop.net/epages/64015097.sf/de_DE/?ObjectPath=/Shops/64015097/Products/DF-000

(Warum hab ich eigentlich auf Englisch geschrieben? 😆 )

**Reply by j3sk0** [reply]
Hallo zusammen, was für eine Tönung könnt ihr empfehlen? Wievielt % sollte es sein? 
 https://www.amazon.de/dp/B07K28R7CJ/ref=cm_sw_r_apan_glt_i_7BMGCRPAWV3T45VM4HRM?psc=1

**Comment by hamster65** [comment]
I simply glued paper (cut from A3 format) onto the grid. This gives a white surface, of course, and the LEDs are very bright. Too bright in fact for many situations, but when I dim them down, the colors drift of or darker pixels go blank. The solution was to reduce the voltage. Still enough for the 3.3V regulator of the ESP. I integrated a step down converter into the PixelIt case.

I also have some diffusor / polarized sheets from old TFTs which I'll try for my second PixelIT. That should give a dark grey reflective surface.

**Comment by Klaus5555** [comment]
Hallo zusammen,
ich möchte mir auch ein PixelIt bauen und iobroker verbinden. Ich bin noch ganz am Anfang. Ich möchte laut Anleitung die Datei PixelIt.bin auf meinem Wemos flashen aber ich finde die Datei nicht. Vielleicht kann mir jemand sagen wo ich die finde. Danke schon mal.

**Reply by o0shojo0o** [reply]
Hi, 
hier findest du alle Releases  https://github.com/o0shojo0o/PixelIt/releases
Für den Wemos ist die **firmware_d1_mini.bin**

**Comment by HansImGlueck43** [comment]
Hallo in die Runde,

vorab: vielen Dank für das schöne Projekt! Auch meine PixelIt ist nun fertig und ich möchte das Ergebnis gern teilen.
Weil ich noch 24mm dicke Multiplex-Bretter liegen hatte, habe ich mir das Gehäuse kurzerhand aus selbigen angefertigt. Das ging - dank vorhandener kleiner Portalfräse und einem 3D-Drucker - ganz gut. So, wie das Gehäuse angelegt ist, könnte man es auch aus jedem anderen Material, beispielsweise aus massivem Aluminium oder Messing fräsen. Würde die Sache noch mal optisch aufwerten, könnte ich mir vorstellen. Die STEP- und STL-Dateien kann ich bei Interesse gern zur Verfügung stellen.
Als Diffusor habe ich auch das bereits erwähnte PERSPEX Frost benutzt. Sieht wirklich klasse aus! Auch die Folie aus dem leuchtbildshop habe ich mir bestellt. Weil die deutlich dünner ist, werden die Pixel viel scharfkantiger. Sieht auch toll aus, aber erfordert einen guten, sauberen Druck des Gitters, weil man jede Macke durch die Folie als Schatten sieht. Das PERSPEX Material verzeiht da einiges.
Falls jemand meinen Entwurf nachbauen möchte, bitte darauf achten, dass die Frästiefen teils angepasst werden müssen. Je nach Dicke der Diffusor-Scheibe oder Folie muss die Tiefe der Fräsung variiert werden.
In die aktuelle Konstruktion sind noch ein paar Änderungen eingeflossen. Es gibt Bohrungen, um in die Frontblende Gewindestangen einkleben zu können.

Hier noch ein paar Bilder:

![IMG_2724](https://user-images.githubusercontent.com/3256969/155674918-bdc558e8-022e-45bb-bcf9-5726d6c354c0.JPG)
![IMG_2728](https://user-images.githubusercontent.com/3256969/155678182-0bb7d998-ce60-4539-a885-e22483740427.JPG)
![IMG_2703](https://user-images.githubusercontent.com/3256969/155678240-69179614-16fc-43b9-a459-32d92f56200e.JPG)
![IMG_2704](https://user-images.githubusercontent.com/3256969/155678268-ee8ef28c-f729-4a4e-92a2-46abd841ff13.JPG)
![IMG_2705](https://user-images.githubusercontent.com/3256969/155678292-85a009c2-9e71-47a2-bf6a-bb81a439dd93.JPG)
![IMG_2706](https://user-images.githubusercontent.com/3256969/155678324-9f7b57b0-ac1b-4e76-946d-9fcb11993938.JPG)
![IMG_2715](https://user-images.githubusercontent.com/3256969/155678352-7f472716-3174-49c5-8864-d49079734d62.JPG)
![PixelIt_3D_01](https://user-images.githubusercontent.com/3256969/155678402-1a09dc16-bbab-4aae-86cb-d5b4600de566.PNG)
![PixelIt_3D_02](https://user-images.githubusercontent.com/3256969/155678415-9cc16ddb-9fe7-4466-becd-66e9bf049d73.PNG)
![PixelIt_3D_03](https://user-images.githubusercontent.com/3256969/155678441-933b0d8c-02c8-4caf-bd19-9788d27f734f.PNG)
![PixelIt_3D_05](https://user-images.githubusercontent.com/3256969/155678460-16d964a0-85de-4813-b9cc-cdb702b247e3.PNG)
![PixelIt_3D_04](https://user-images.githubusercontent.com/3256969/155678470-85d7a83b-9f16-44d3-a5da-51fa27b2d0ec.PNG)

**Reply by o0shojo0o** [reply]
Wow, echt toll geworden!

**Reply by miccgn** [reply]
Hallo Hans,
> Auch die Folie aus dem leuchtbildshop habe ich mir bestellt. Weil die deutlich dünner ist, werden die Pixel viel scharfkantiger. Sieht auch toll aus, aber erfordert einen guten, sauberen Druck des Gitters, weil man jede Macke durch die Folie als Schatten sieht. Das PERSPEX Material verzeiht da einiges. 

ich nehme einfach beides! Die Folie ist direkt vor dem Gitter, dann kommt erst das Perspex. Probier das ruhig mal aus - die Folie trägt ja nicht viel auf.

Das Gehäuse sieht mega aus! Ich habe bislang nur 3D-Drucke in Auftrag gegeben.

**Comment by HansImGlueck43** [comment]
Danke!
Hier dann auch die STEP-Dateien. Darin enthalten auch die mit Bohrungen versehenen originalen GRID-Dateien...

[PixelIt - HansImGlueck43.zip](https://github.com/o0shojo0o/PixelIt/files/8140012/PixelIt.-.HansImGlueck43.zip)

**Reply by dolomiti70** [reply]
Dein Gehäuse sieht super aus. Bei mir kommt immer "Not found" wenn ich das Zip-File runterladen will.

**Reply by HansImGlueck43** [reply]
[PixelIt - HansImGlueck43.zip](https://github.com/pixelit-project/PixelIt/files/11484521/PixelIt.-.HansImGlueck43.zip)

Falls du das Gehäuse nachbaust, würde ich mich freuen, wenn Du dein Ergebnis auch hier vorstellst...

**Comment by NatroNx** [comment]
![20220703_125502138_iOS](https://user-images.githubusercontent.com/10463761/177055832-3f53bc0a-9dc7-4d82-9be9-9ed7435a8294.jpg)

I'm not sure what wood that is, but I know that it is at least 60 years old. I like the combination of old and modern

**Comment by NatroNx** [comment]
und der kleine Bruder ist nun auch fast fertig.

Mit versteckten Tastern wieder aus Holz und Plexiglas

![20220710_210656584_iOS](https://user-images.githubusercontent.com/10463761/178162708-e9ede651-a463-4e6b-8203-08260667a3c6.jpg)

![20220710_210752914_iOS](https://user-images.githubusercontent.com/10463761/178162711-46ce4c16-bb93-4b89-b68d-26b101871d16.jpg)

**Comment by d4rkd3v1l** [comment]
Ich lass das erstmal so^^

Thingiverse Gehäuse (PLA schwarz): https://www.thingiverse.com/thing:3559014
Vorne Acrylglas mit Milchglasfolie drauf: https://expresszuschnitt.de/Acrylglas-satiniert-grau und https://kreativmanufaktur.bayern/epages/f41caa8a-1aea-495f-91d6-ef9485e1db47.sf/sec57ce0e68f0/?ObjectPath=/Shops/f41caa8a-1aea-495f-91d6-ef9485e1db47/Products/Milchglasfolie/SubProducts/Milchglasfolie_30x60

![IMG_1142](https://user-images.githubusercontent.com/13336238/196218752-ff6db781-7321-4340-9953-6984dfbbd845.jpeg)

**Reply by iz3man** [reply]
Ist diese Milchglassfolie AUSSEN? Oder innen? Muss wohl innen sein, sonst hättest du kein satiniertes Plexiplas genommen, oder?

**Comment by Big-Brummer** [comment]
Hallo und vielen Dank für dieses tolle Projekt.
Hier meine Realisierung:
Ich habe das Gehäuse aus einem massiven Holzstück gefräst.
Für die Scheibe habe ich diese verwendet: https://www.plattenzuschnitt24.de/Acrylglas--getoent-Schwarz-3mm.html
Als Diffusor Folie habe ich diese verwendet: https://www.folien-fischer-shop.de/diffusorfolien-led-filterfolien/2999/streufolie/diffusorfolie-30-lichtdurchlass-rollenbreite-123-cm
ich bin nicht ganz mit der Schärfe der pixels zufrieden.
Sie sind etwas verwaschen und unscharf.
Wie dick sind denn Eure Gitter ? ich habe 3mm gewählt. ist das zu dick oder zu dünn ?
![20230830_170643 1](https://github.com/pixelit-project/PixelIt/assets/24531493/ecde8db2-063d-443f-a311-51102db15107)

**Reply by foorschtbar** [reply]
Wow, sieht richtig schick aus. 

Das unscharfe kommt IHMO von der getönten Scheibe. Ich habe auch eine 3mm Acryl Scheibe, aber eine klare. Als Defusor habe Butterbrotpapier genommen und damit sind die Pixel sehr scharf.

**Reply by iz3man** [reply]
Ich denke, dass das Unscharfe eigentlich gewollt ist, weil sonst könnte man ja den Diffusor weglassen, oder?

**Reply by NatroNx** [reply]
wenn die Kanten der Pixel unscharf sind, ist der diffusor zu dick oder das Gitter liegt nicht 100% auf diffusor / scheibe auf. Hatte ich auch. Butterbrotpapier geht echt super - und sehr günstig

In denem fall sieht es aber super aus, finde ich

**Comment by Webranger2** [comment]
<img width="548" alt="Bildschirmfoto 2023-10-27 um 16 13 10" src="https://github.com/pixelit-project/PixelIt/assets/44174579/5bf24398-5d34-4985-b7f5-f8a280c9a2a7">

War mal eine Awtrix bin aber jetzt umgestiegen auf PixelIt.

**Comment by physics777777** [comment]
Hallo zusammen,
ich bin auch interessiert mir eine solche Matrix mit pixelit zu bauen. Ich komme aus dem Autodarts-Bereich. Jetzt stehe ich direkt vor einem Problem. Ich habe bereits eine 8x32 Matrix und zwei Controller hier rumliegen und würde gerne wissen, ob ich das Projekt (Pixelit) mit einem der Controller starten kann (habe auch noch ein Netzteil hier und wäre froh, wenn es mit dem Netzteil gehen würde, da ich in dem Bereich ungerne löten möchte.)
Hier die passenden Links zu den Materialien:
- https://de.athom.tech/blank-1/wled-rgbcct-analog-and-digital-light-strip-controller
- https://www.amazon.de/dp/B07DVJ568C?psc=1&ref=ppx_yo2ov_dt_b_product_details
- https://www.amazon.de/dp/B088K1KDW5?psc=1&ref=ppx_yo2ov_dt_b_product_details
- https://www.amazon.de/dp/B088FBXTPS?psc=1&ref=ppx_yo2ov_dt_b_product_details

Vielen Dank für eure Hilfe!

**Comment by miyumo82** [comment]
I want to build big pixel clock. And I start with WS2812B Led and PixelIt Firmware

Link for 3d print file:[](https://www.printables.com/model/1052824-large-pixelit-clock)

https://github.com/user-attachments/assets/0bef8bff-bed5-40c2-a6d6-af18b4522dfd

![IMG_8388](https://github.com/user-attachments/assets/48a3368b-8660-4431-8e09-1fc9bcf4e159)
![Led Lock - 60 v70-2](https://github.com/user-attachments/assets/d081bb7a-c2de-490b-9074-5da46fe53745)

**Reply by daniel4999** [reply]
You can avoid the LED bleeding by using a dark color (black) inner grid

**Comment by chrisper0815** [comment]
![20241231_131409](https://github.com/user-attachments/assets/60b6e7e5-b864-4425-ab1a-a1d66c08ac39)
![20241231_131351](https://github.com/user-attachments/assets/0c653c57-5c78-4799-81dd-732633869955)
 3d printed, only with 3 button and 4* 8x8 LED matrix

**Reply by iRaschen** [reply]
> ![20241231_131409](https://private-user-images.githubusercontent.com/89187925/399470343-60b6e7e5-b864-4425-ab1a-a1d66c08ac39.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzg3NDgwOTYsIm5iZiI6MTczODc0Nzc5NiwicGF0aCI6Ii84OTE4NzkyNS8zOTk0NzAzNDMtNjBiNmU3ZTUtYjg2NC00NDI1LWFiMWEtYTFkNjZjMDhhYzM5LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA1VDA5Mjk1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlNzU3NzIwOGRhODUyYjY2MzUwMmM1YWRkOTAzNjdlYWJlNGVhMzAwNzczYmMyOTMzNmYzZDg2M2Y3YWVjYjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Bp0HMSTVyd6wxm91Xup1yqM94IhnTpIshkVqPoC5MXo) ![20241231_131351](https://private-user-images.githubusercontent.com/89187925/399470285-0c653c57-5c78-4799-81dd-732633869955.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzg3NDgwOTYsIm5iZiI6MTczODc0Nzc5NiwicGF0aCI6Ii84OTE4NzkyNS8zOTk0NzAyODUtMGM2NTNjNTctNWM3OC00Nzk5LTgxZGQtNzMyNjMzODY5OTU1LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA1VDA5Mjk1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3MmMyZWRiNzhlOGM2MzVkZjdkOTk0NDY1OWNhY2Q0NmM2MmFiYWY2OWIyMjhiZDIzMDg0NTBhYmRmZmQyZjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.TYrHtr1-WiZs1FzDtyOIxgmVXBJPFHieE6fmg7CoeY8) 3d printed, only with 3 button and 4* 8x8 LED matrix

where can i download stl or fusion 360 project?

**Comment by m-boy25** [comment]
![unnamed](https://github.com/user-attachments/assets/9705cb64-ba11-4f03-b96a-ab9534ccddc1)
i dont found other material because i dont have a 3d printer
![pixel it](https://github.com/user-attachments/assets/936b4e30-12fb-42ee-ac8f-5de9172c5779)
![pixelit](https://github.com/user-attachments/assets/e62d9fa3-d21b-4e91-a205-0bffe7894b9b)   
i dont found diffuser for the matrix but alway works the button are fine i just have the minimum

**Reply by daniel4999** [reply]
You can use a semi transparent document folder as a diffuser.
![Screenshot 2025-01-17 231447](https://github.com/user-attachments/assets/61707c54-8575-4654-886a-0c6bc28e5eab)

**Comment by MasterChief1335** [comment]
Hi ich habe BME680 Sensor  Angeschlossen aber bekomm nix angezeigt?

**Comment by MikeKMiller** [comment]
I embedded one in my office door. It auto switches to an "On A Call Message" if I am on the phone, or Zoom via HomeAssistant
![20251211_133038~2](https://github.com/user-attachments/assets/13dd18ff-ee73-498f-b886-1b08f1a36e97)
